### PR TITLE
Multi inout grucell

### PR DESCRIPTION
**Pull request by lhs8928**

### CODE REVIEW

File: nntrainer/layers/zoneout_lstmcell.cpp [hunk 1]
```diff
@@ -95,27 +91,51 @@ void ZoneoutLSTMCellLayer::finalize(InitLayerContext &context) {
...
-  if (context.getNumInputs() != 1)
-    throw std::invalid_argument("ZoneoutLSTMCellLayer takes only one input");
-  if (std::get<props::MaxTimestep>(zoneout_lstmcell_props).empty())
-    throw std::invalid_argument("Number of unroll steps(max timestep) must be "
-                                "provided to zoneout LSTM cells");
-  if (std::get<props::Timestep>(zoneout_lstmcell_props).empty())
+  if (context.getNumInputs() != 3) {
     throw std::invalid_argument(
-      "Current timestep must be provided to zoneout LSTM cell");
+      "Number of input is not 3. ZoneoutLSTMCellLayer should takes 3 inputs");
```
Comment (jijoongmoon) [inline nntrainer/layers/zoneout_lstmcell.cpp:96]: Could you leave the comment what 3 inputs are?

File: nntrainer/layers/zoneout_lstmcell.cpp [hunk 2]
```diff
@@ -343,15 +330,14 @@ void ZoneoutLSTMCellLayer::forwarding(RunLayerContext &context, bool training) {
     prev_cell_state.multiply(prev_cell_state_zoneout_mask, cell_state, 1.0f);
   }
   // Todo: zoneout at inference
-
-  output.copyData(hidden_state);
```
Comment (jijoongmoon) [inline nntrainer/layers/zoneout_lstmcell.cpp:347]: I think it is good at least.

**Review by jijoongmoon — APPROVED** [review]
overall LGTM except for CI

File: nntrainer/layers/grucell.cpp [hunk 3]
```diff
@@ -37,17 +37,226 @@
 
 #include <layer_context.h>
 
+#include <iostream>
+
+#define ppline \
+  std::cout << "[" << __FILE__ << "]" << __func__ << ":" << __LINE__ << "\n";
+#define pp(x) std::cout << #x << "\n" << x << "\n";
+
```
Comment (zhoonit) [inline nntrainer/layers/grucell.cpp:45]: this should be deleted.

**Review by zhoonit — APPROVED** [review]
LGTM after CI pass :) Thank you!

**Pull request merged**